File: frontend/src/entities/comments/api/dto/comment.dto.ts
import { Account } from "@/entities/account/model/account";

export interface CommentDto {
    tlgId: number;
    peerId: number;
    postId: number;
    postTlgId: number;
    from: Account;
    message: string;
    date: Date;
    replyTo: number;
}

File: backend/services/GatherService/Dtos/CommentDto.cs
using Gather.Models;

namespace Gather.Dtos;

public class CommentDto
{
    public int Id { get; set; }

    public long TlgId { get; set; }

    public long PeerId { get; set; }

    public long PostId { get; set; }

    public Account? From { get; set; }

    public string Message { get; set; } = string.Empty;

    public DateTime Date { get; set; }

}
